app/workers/send_email_worker.rb

```diff
@@ -13,15 +13,15 @@ class DeliveryRequestWorker < ApplicationWorker
   def perform(email_id, metrics, queue)
     if rate_limit_exceeded?
       logger.warn("Rescheduling email #{email_id} due to exceeding rate limit")
-      GovukStatsd.increment("delivery_request_worker.rescheduled")
-      DeliveryRequestWorker.set(queue: queue || "delivery_immediate")
+      GovukStatsd.increment("send_email_worker.rescheduled")
```

Comment:
Minor: we don't use this, and it's inconsistent with the (apparent) pattern of proxying metrics through [the monolith](https://github.com/alphagov/email-alert-api/blob/master/lib/metrics.rb). Since we're renaming it, we could just remove it.

Reply:
Sure, removed.